stubs/ORM/Query/Expr.stub

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace Doctrine\ORM\Query;
+
+class Expr
+{
+
+    /**
+     * @return ($x is literal-string ? literal-string&non-empty-string : string)
+     */
+    public function isNull($x)
+    {
+    }
+
+    /**
+     * @return ($x is literal-string ? literal-string&non-empty-string : string)
+     */
+    public function isNotNull($x)
+    {
+    }
+
+    // /**
+    //  * @return ($x is array<literal-string> ? literal-string&non-empty-string : string)
+    //  */
+    // public function countDistinct($x)
+    // public function countDistinct(...$x)
+    // {
+    // }
+
+    // /**
+    //  * @return ($val is literal-string AND $x is literal-string AND $y is literal-string ? literal-string&non-empty-string : string)
```

Comment:
You can follow this example
```
/**
 * @param int|float $a
 * @param int|float $b
 * @return ($a is int ? ($b is int ? int : float) : float)
 */
function add($a, $b) {
    return $a + $b;
}
```

I don't know if AND is supported

Reply:
Thank you @VincentLanglet, that's exactly what I needed.

If I'm reading the [TypeParser](https://github.com/phpstan/phpdoc-parser/blob/129a63b3bc7caeb593c224c41f420675e63cfefc/src/Parser/TypeParser.php#L251) correctly (when it's doing a [subParse()](https://github.com/phpstan/phpdoc-parser/blob/129a63b3bc7caeb593c224c41f420675e63cfefc/src/Parser/TypeParser.php#L43), due to there being a [TOKEN_OPEN_PARENTHESES](https://github.com/phpstan/phpdoc-parser/blob/129a63b3bc7caeb593c224c41f420675e63cfefc/src/Parser/TypeParser.php#L75)), and checking the [ConditionalTypeForParameterNode](https://github.com/phpstan/phpdoc-parser/blob/1.4.x/src/Ast/Type/ConditionalTypeForParameterNode.php) object, then conditional return types use the format `(%s %s %s ? %s : %s)` where the second `%s` is either "is" or "is not" - which implies there is no support for operators like AND.